R/cluster.R

```diff
@@ -220,10 +241,11 @@ makeCluster <-
         fill = TRUE
       )
 
-
       clusterNodeMismatchWarning <-
-        paste0("There is a mismatched between the projected cluster %s ",
-               "nodes min/max '%s'/'%s' and the existing cluster %s nodes '%s'")
+        paste0(
+          "There is a mismatched between the projected cluster %s ",
```

Comment:
Here is one example where we use 'cluster' and not 'pool'.

Reply:
Fixed.